### DISCUSSION TITLE
Can't display posts.

### DISCUSSION THREAD

**Discussion by yjlc-pc** [cotes2020/jekyll-theme-chirpy#2138] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I created a repository:[yjlc-pc.github.io](https://github.com/yjlc-pc/yjlc-pc.github.io) 
I wanted to compose a test blog and when I put it in the /_posts folder, it didn't show up on the site.

### Operations you have already tried

I tried changing ./config.yml, but it didn't work.

**Comment by SoumyaK4** [comment]
@yjlc-pc enter a space after line 6 [here](https://github.com/yjlc-pc/yjlc-pc.github.io/blob/main/_posts/test.md?plain=1)

**Reply by yjlc-pc** [reply]
[file](https://github.com/yjlc-pc/yjlc-pc.github.io/blob/main/_posts/test.md)
Is that so?

**Reply by SoumyaK4** [reply, marked as answer]
@yjlc-pc yes it should've worked but isn't working....
If I remember correctly the tags should be in lower case, can you do that and check please..


Edit: the files in post should be named like this
`yyyy-mm-dd-post-name.md`

Here's a [demo](https://github.com/BB-Dojo/weiqi/blob/main/_posts/2024-10-01-What's-This-All-About.md?plain=1)
